src/utils/index.ts

```diff
@@ -0,0 +1,4 @@
+export const getRandomItem = <T>(arr: T[]) => {
```

Comment:
perhaps pull out the "shuffle" function  in `src/util/iterators.ts` and make a new file `src/util/arrays.ts` with that and this one

Reply:
Oooo, I love that idea! I've got a few other utility functions that I find myself reusing from ML project to ML project, great idea to have a separate utility folder for these general purpose, reusable utility functions that we expose to make life easier when building around shumai.

Reply:
Will get this done in the next commit as well.